docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -36,109 +41,118 @@ $ sudo snap install edgexfoundry --channel=2.0
 !!! Warning
     Running the EdgeX snap on a machine setup for EdgeX development can create conflicts and result in the platform errors/issues.
 
-## Using the EdgeX snap
+## Using the EdgeX Snap
 
 Upon installation, the following EdgeX services are automatically and immediately started:
 
     - consul
+    - vault
     - redis
+    - kong
+    - postgres
     - core-data
     - core-command
     - core-metadata
-    - security-services (see note below)
+    - security-services (see [Security Services section](#security-services-section) below)
 
 The following services are disabled by default:
 
     - app-service-configurable (required for eKuiper)
     - device-virtual
     - kuiper
-    - support-logging
     - support-notifications
-    - support-rulesengine (deprecated)
     - support-scheduler
     - sys-mgmt-agent
```

Comment:
Remove indentation. These are recognized as code blocks in the rendered version.

Reply:
Removed.